k8s/wrappers/services/k8s-apiserver-proxy

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+
+. "$SNAP/k8s/lib.sh"
+
+k8s::common::execute_service k8s-apiserver-proxy
```

Comment:
^ NOTE that we extract the proxy into a separate `k8s-apiserver-proxy` entrypoint so that we can do this